src/Avalonia.Base/Utilities/WeakSubscriptionManager.cs

```diff
@@ -22,10 +22,11 @@ public static class WeakSubscriptionManager
         public static void Subscribe<TTarget, TEventArgs>(TTarget target, string eventName, IWeakSubscriber<TEventArgs> subscriber)
             where TEventArgs : EventArgs
         {
+            _ = target ?? throw new ArgumentNullException(nameof(target));
```

Comment:
I don't know how to feel about this syntax tbh.
Looks good enough to me for now. I think we will need to have another pass with formatting tool as we planned before, and decide on it.